internal/gitserver/gitolite.go

```diff
@@ -0,0 +1,50 @@
+package gitserver
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/url"
+
+	"github.com/sourcegraph/sourcegraph/internal/conf"
+	"github.com/sourcegraph/sourcegraph/internal/extsvc/gitolite"
+	"github.com/sourcegraph/sourcegraph/internal/httpcli"
+)
+
+type GitoliteLister struct {
+	addrs func() []string
+	cli   httpcli.Doer
```

Comment:
```suggestion
	httpClient   httpcli.Doer
```

for the sake of naming consistency (at least in `gitserver` package)